### DISCUSSION TITLE
How about add a new portal about the Window Taskbar Progress?

### DISCUSSION THREAD

**Discussion by SnowNF** [flatpak/xdg-desktop-portal#1236] [Ideas]
Based on [Wayland protocol issue comment](https://gitlab.freedesktop.org/wayland/wayland-protocols/-/issues/168#note_2191593)   [GLFW Taskbas Progress PR](https://github.com/glfw/glfw/pull/2286) and discussion on Matrix.

It seems that only KDE and the Unity desktop environment utilize this API. Can we add a new XDG Portal to implement this better?

I'm sorry that I have no experience in XDG Portal.

**Comment by TingPing** [comment]
Can you link to specs for Unity's API (which I assume KDE uses)?

**Reply by TingPing** [reply]
Ah I see https://wiki.ubuntu.com/Unity/LauncherAPI in the first link.

**Comment by Mikenux** [comment]
On GNOME, progress notifications have been discussed. Since the progression UI is supposed to be independent of the information, couldn't this portal be linked to it?

**Reply by razzeee** [reply]
If you mean not showing progress on the app icon but in a different place, yes this should allow for that and IMO would be the best design for stock gnome.

**Reply by Mikenux** [reply]
Thanks!

@jsparber: Are you also working on progress notifications? If so, do you think this work could be adapted to what is requested here?
